src/credentials/FabricTable.cpp

```diff
@@ -335,6 +341,20 @@ CHIP_ERROR FabricInfo::SetOperationalKeypair(const P256Keypair * keyPair)
     return mOperationalKey->Deserialize(serialized);
 }
 
+CHIP_ERROR FabricInfo::SetExternallyOwnedOperationalKeypair(P256Keypair * keyPair)
```

Comment:
As a follow-up, maybe some of the keypair methods should become const, so we can take a `const P256Keypair *` here?

Reply:
Sounds good